build/scripts/build-volplugin-containers.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+
+# In order for the new binaries to take effect we need to clean the older environment
+# Otherwise the binaries stick to older container images
+# repeated docker builds also lead to dangling images, which need to be cleaned too.
+docker ps -a | grep -e volplugin -e apiserver -e volsupervisor | awk '{print $1}' | xargs docker rm -fv
+docker rmi contiv/volplugin
+docker rmi $(docker images -f "dangling=true" -q)
```

Comment:
I've read that using `dangling=true` can lead to some destructive behavior. Are you familiar with this at all?